testsuite/classlibrary/TestFunction.sc

```diff
@@ -66,6 +72,8 @@ CommonTestClass {
 		}
 	}
 
+	*reset { x = nil; y = 7; }
+	reset { a = nil; b = 3; }
```

Comment:
i understand, but if you actually put them in `setUp`, they'd be called automatically before the test, so there would be no need for code duplication.